### PR TITLE
Bump rubygem-foreman_rh_cloud to 5.0.29-1

### PR DESCRIPTION
**Pull request by ShimShtein**

### CODE REVIEW

File: packages/katello/rubygem-foreman_rh_cloud/rubygem-foreman_rh_cloud.spec [hunk 1]
```diff
@@ -48,16 +48,13 @@ Provides: foreman-plugin-%{plugin_name} = %{version}
 # start package.json devDependencies BuildRequires
 BuildRequires: %{?scl_prefix}npm(@babel/core) >= 7.7.0
 BuildRequires: %{?scl_prefix}npm(@babel/core) < 7.8.0
-BuildRequires: %{?scl_prefix}npm(@redhat-cloud-services/frontend-components) >= 2.5.0
-BuildRequires: %{?scl_prefix}npm(@redhat-cloud-services/frontend-components) < 3.0.0
```
Comment (evgeni) [inline packages/katello/rubygem-foreman_rh_cloud/rubygem-foreman_rh_cloud.spec:52]: this breaks it.

```
ERROR in /builddir/build/BUILDROOT/rubygem-foreman_rh_cloud-5.0.29-1.fm3_2.el8.noarch/usr/share/gems/gems/foreman_rh_cloud-5.0.29/webpack/InsightsCloudSync/Components/InsightsTable/table.scss
Module build failed: ModuleNotFoundError: Module not found: Error: Can't resolve '@redhat-cloud-services/frontend-components/index.css' in '/builddir/build/BUILDROOT/rubygem-foreman_rh_cloud-5.0.29-1.fm3_2.el8.noarch/usr/share/gems/gems/foreman_rh_cloud-5.0.29/webpack/InsightsCloudSync/Components/InsightsTable'
```

Comment (ShimShtein) [reply packages/katello/rubygem-foreman_rh_cloud/rubygem-foreman_rh_cloud.spec:52]: It's strange, I do see `./packages/foreman/nodejs-redhat-cloud-services-frontend-components/nodejs-redhat-cloud-services-frontend-components-2.5.0`. And the version looks correct.

Comment (evgeni) [reply packages/katello/rubygem-foreman_rh_cloud/rubygem-foreman_rh_cloud.spec:52]: Sure, but you removed the BuildRequires, so nothing pulls it in.

File: packages/katello/rubygem-foreman_rh_cloud/rubygem-foreman_rh_cloud.spec [hunk 2]
```diff
@@ -50,14 +50,13 @@ BuildRequires: %{?scl_prefix}npm(@babel/core) >= 7.7.0
 BuildRequires: %{?scl_prefix}npm(@babel/core) < 7.8.0
 BuildRequires: %{?scl_prefix}npm(@redhat-cloud-services/frontend-components) >= 2.5.0
 BuildRequires: %{?scl_prefix}npm(@redhat-cloud-services/frontend-components) < 3.0.0
```
Comment (Ron-Lavi) [inline packages/katello/rubygem-foreman_rh_cloud/rubygem-foreman_rh_cloud.spec:52]: This one can be removed as v3.3.15 is supported by forman-js: https://github.com/theforeman/foreman-js/blob/master/packages/vendor-core/package.json#L35

File: packages/katello/rubygem-foreman_rh_cloud/rubygem-foreman_rh_cloud.spec [hunk 3]
```diff
@@ -50,14 +50,13 @@ BuildRequires: %{?scl_prefix}npm(@babel/core) >= 7.7.0
...
-BuildRequires: %{?scl_prefix}npm(@theforeman/builder) >= 4.14.0
+BuildRequires: %{?scl_prefix}npm(@theforeman/builder) >= 8.16.0
+BuildRequires: %{?scl_prefix}npm(@theforeman/builder) < 8.17.0
```
Comment (Ron-Lavi) [inline packages/katello/rubygem-foreman_rh_cloud/rubygem-foreman_rh_cloud.spec:54]: you can remove the upper border open as we used to handle it before, e.g:

```suggestion
BuildRequires: %{?scl_prefix}npm(@theforeman/builder) >= 8.16.0
```

**Comment by ShimShtein** [comment]
@evgeni I suppose we can continue with this dependency and remove it in the next version.

File: packages/katello/rubygem-foreman_rh_cloud/rubygem-foreman_rh_cloud.spec (hunk 3 above)
Comment (evgeni) [reply packages/katello/rubygem-foreman_rh_cloud/rubygem-foreman_rh_cloud.spec:54]: @ShimShtein this is still open

**Review by evgeni — APPROVED** [review]

**Pull request merged**